Source/Lib/Codec/EbPictureManagerProcess.c

```diff
@@ -503,7 +503,7 @@ void* PictureManagerKernel(void *inputPtr)
 				   pictureControlSetPtr->referencePictureWrapperPtr = (EbObjectWrapper_t*)EB_NULL;
 			   }
 #if DEADLOCK_DEBUG
-               SVT_LOG("POC %lld PM OUT \n", pictureControlSetPtr->pictureNumber);
+               SVT_LOG("POC %lu PM OUT \n", pictureControlSetPtr->pictureNumber);
 #endif
```

Comment:
why it is here?
should be after postfullobject?

Reply:
Thanks! Resolved in the 2nd commit.